src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react'
+import { Route, Routes }  from 'react-router-dom'
+import './App.css'
+
+import Header from './components/header/header'
+import Menu from './components/menu/menu'
+import Content from './components/content/content'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <>
+    <div className="container">
+      <Header/>
+      <Menu/>
+      <Routes>
+        <Route
+        path='/'
+        element={<Content/>}
+        />
+      </Routes>
+    </div>
```

Comment:
Structure looks good! Components all look to be named sensibly and look to be in the correct places. Folder structure is logical to follow.